workflows/offline_ml_diags/offline_ml_diags/_helpers.py

```diff
@@ -46,6 +47,52 @@
 ScalarMetrics = Dict[str, Mapping[str, float]]
 
 
+def _count_features_2d(
+    quantity_names: Iterable[str], dataset: xr.Dataset, sample_dim_name: str
+) -> Mapping[str, int]:
+    """
+    count features for (sample[, z]) arrays.
+    Copied from fv3fit._shared.packer, as this logic is pretty robust.
+    """
+    for name in quantity_names:
+        if len(dataset[name].dims) > 2:
+            value = dataset[name]
+            raise ValueError(
+                "can only pack 1D/2D (sample[, z]) "
```

Comment:
I'd consider switching "pack" to "count" for these errors since it might be confusing w/ no packing happening.